api/bq-test-key.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "all-of-us-workbench-test",
+  "private_key_id": "49e6274976403f169ea708e4eeca2e0f57998543",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQC5KeRb5jKGFRwY\nsdJf1QiPkSTeYfUM9WywE7kBySG531W6wKNimhsoII+Qg6VNc7vXaHBGC+GuEkeY\nRVloVTMl+xc/i4ieCvaLkxIuoszas7zuqe26j/KhJwOcZXk+5IxqThjg4tPhEnRw\nEAZ2fJHJo1e9uUbyB3f81Nd+Wb2H+ZVoNuEeVspzI+NXaAR4OIrjSpF7axzPWP8M\nksX0HN4rQVOhyT4Ur9cN3DWBbImEdI+1YJs/cdNhwLOwTsJu7SwHWY3QWovlx7TG\nasnVN3IWGow+m8mbiHltA/kVjQmkNloBcYy5OAFuyZCZxJ+ffs1mGcgc2qP3oQQc\n5C2bm7mZAgMBAAECggEAUFErFG/cT4vdlZqgmeAS6A958hqEjJvyisfrUqp3fiQx\nkCHaXT1idTv2eiWYcOm4O546QTALMxILyzGdsCNw3dM2COLSHle8DNv9IYyNlnor\nhEziaS/ie2Osbc0YVB8YB06b/QR3CqSSLD/PRdsP8+pt90ZkTzzNH7cFSHHNxCpR\nbim/kRUXDexD8fu5UV+20+LV/zMa0XYatPGxMekNLw5Y/GS6W4O+7nqOWfXo0yiY\nbTxIcjHQ+e2ys1vUv93CM47kKOp73ifmFhDT4XkGp/fYtBg2r5hBIxQ5xjWlvaHU\nylCeCVutdw67nq+aZE7vJ3LIQe7GYAx10PjKZvWQnwKBgQDxhrjIDHd795zbfBvI\nBS4UdOned0DJndwcut8IDax9ArdQSaRkDG/3KuDkAmmFhpsy6ETaVBenA1a0RGa+\n3PiWBobsDgaj9vAdeuNV4HZ4Fi4Ow+4gouTt0jGFSeiiI4GNpl6RQLFYK/KfvMTM\nN8yFO2FYxHfsXxrzkD0BeCvM6wKBgQDEQoF5DhWJIBavecpqpWFs9OTWB1/0zZCj\n66ijDeJ/yN2njhIWNVjyK4ZIGngS4aCBoVKLi3TM9rmHBz1TRlwcaz9R9bTh2BZz\nZT1+DGJ9uckVgiBlFgYdr9G/KXX2zp646W+wepYZFB41Lb+YJotOmQcZdYtw79GU\nkZUeJ8TiiwKBgQCn1JnqSN9uQS26Ro93ekH+7Q7mC6/1acBeJmx8yCOn6o8/Ny0I\naKwU8LK9Pf/dTR/jTAssO2sEHwnM0gsjbaE/fuGaflSW8yz/DzmRnxwzbhYrJHMA\nMrEpoPnQSP9iS5HmwEjOXkndnwxFe4K09cEkg5KtgDhlA+FShlQeBUdKaQKBgAed\niQaYfE9w1lCCyk7vEeO3cPW5HYkTPOyPZWDGViadNbSdt6fy8/uZosFDfatF5AOD\nrf9KLwMRA8vwhPmpPkPACZn1jcpBp0VdBVDRzDSSs/HItdcWTrByNRJU2oX0KIaG\nUFYpu49Hj5CCsO+Hd1paZ8MeI4YDWD+Kkp2z71jXAoGBAIo3OxnOLGxrUwbuDJwi\nF2IPgURXHV5STVhqhHqUpadA+uZdWiTINA+K4drGAw6YM9bJGVhykisKRqYIo39B\nc2Cv3DpApJpb+AmjtUoA2Htgz5mVyiHmOyGjVQrlEojMBulU48APMXt7GQhGrs4P\nlxeM8OTnDI4g9ruA39UfYcR8\n-----END PRIVATE KEY-----\n",
+  "client_email": "bigqueryservice@all-of-us-workbench-test.iam.gserviceaccount.com",
```

Comment:
If we're going to check this in, we should make sure this SA has access *only* to BigQuery.

The alternative would be to make the user run gcloud auth login before running tests to create an SA file, like we do for the server. But that will definitely be more of a pain...

Reply:
Sure, that makes complete sense. I'm not sure to tell what SA's have access to what? Do we currently have an SA with only access to BigQuery?

Reply:
I will make this SA only able to access BigQuery... let me know when you're ready to test.